docs/variants/qemu_q35/building-manual.md

```diff
@@ -46,157 +46,20 @@ Update the submodules in order get latest dependencies.
 git submodule update --init --checkout --recursive
 ```
```

Comment:
I think this can be dropped as well, I was able to build fine without updating modules explicitly.

Reply:
https://github.com/Dasharo/docs/compare/2a47bdd5f3eae81d35cc9545010489747bac76c6..fd1fc06fa12f4d9dde6a66b01a567657c6bb1a5d